app/models/page.rb

```diff
@@ -46,10 +46,6 @@ def show_selection_options
     answer_settings.selection_options.map { |option| option.attributes[:name] }.join(", ")
   end
 
-  def submit
```

Comment:
This is no longer used (and possibly never was) but it seemed like a good time to junk it.